### DISCUSSION TITLE
Hook doesn't be executed when processing backup.

### DISCUSSION THREAD

**Discussion by makabaka-boom** [vmware-tanzu/velero#3269] [Community support Q&A] [answered]
velero version : 1.5.2

Hi, I want to use velero to backup application data rather than application. So I set `includeResouces==persistentvolumeclaims,persistentvolumes` and a hook to the pod that used those pvs. Then the hook didn't be executed. It seems like that I misunderstand the scope of hooks. I think maybe hook should be available for entire backup rather than single pod. It is useful for only backing up application data.

**Comment by dsu-igeek** [comment, marked as answer]
I added this issue to track this: https://github.com/vmware-tanzu/velero/issues/3542 It would be a nice feature but there are a few tricks with regards to where to execute the hooks and security concerns.
